bin/_mkproject.bat

```diff
@@ -55,9 +65,10 @@ for %%i in (*.bat) do (
     set temp=%%i
     echo   !temp:.bat=!
 )
+endlocal
 
 echo+
-echo   1. Type first letters of asset
+echo   1. Type first letters of asset category
```

Comment:
Let's remove all notions of a category, then I think we're finished with this PR.

I'll mark each for completeness.